### ISSUE TITLE
Are the weights provided in the repo trained weights?

### ISSUE THREAD

**Issue by shayan09** [aigagror/ml_aesthetics#17]
On running the main.py, I am getting all the results around 5 (for good and bad images).

**Comment by huangeddie** [comment]
Yeah. 5 is quite a common score. Maybe try using more varied images

**Comment by shayan09** [comment]
I tried really bad quality images as well(blurred, pixelated, etc.) but the score hovers around 5 - 5.5, maybe I am not able to understand the criteria for evaluation or it isn't trained enough.

**Comment by sanchit88** [comment]
Hi, I have also found that same problem. It seems like there is a problem with the model.
I have tried it in too bad (very blurred) or quite good look and all it outputs around 5 scores.
